two-phase-commit/src/Makefile

```diff
@@ -0,0 +1,35 @@
+BIN := bin/
```

Comment:
In our other Makefiles the trailing slash is omitted in such declarations and instead inserted later as needed. This makes parsing subsequent paths easier by providing familiar directory boundaries. It also helps prevent double slashes (e.g., `bin//`) from appearing such as in the `clean` rule.

Reply:
fixed